_posts/2020-02-28-status-update.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: post
+title: "28th Feb 2020: IAC Team Status Update"
+author: michaeltlombardi
+categories:
+  - team
+  - status
+tags:
+  - module updates
+  - community
+  - litmus
+---
+
+The IAC team has been working on a lot of interesting things over the past week.
+Here's a brief overview of what we've been up to:
+
+## Updates to Puppet Modules
+
+No modules were released since last week.
+
+## Community Contributions
+
+Thanks to [@jmangt](https://github.com/jmangt) for their improvements to the provision module,
+[adding support for sssh'ing to vagrant boxes where root is not allowed to ssh](https://github.com/puppetlabs/provision/pull/103) and [ensuring the vagrant provision task works against paths with spaces in them](https://github.com/puppetlabs/provision/pull/101).
+
+<!-- ## Approved Modules
```

Comment:
I don't think any new modules were approved? If not, I can delete this section entirely.